### PR TITLE
Telemetry Teaser finished

### PR DESCRIPTION
**Pull request by brycecayanan**

### CODE REVIEW

File: Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html [hunk 1]
```diff
@@ -4,15 +4,31 @@ <h1> Elysia Range Simulator </h1>
...
+      <div *ngIf="batteryError == 1; else batteryError2" class="errorMessage">Battery percentage is required</div>
+      <br/>
       <img src="/assets/Cloud.png">
       <input type="range" min="0" max="100" value="50" (mouseup)="onMouseWeather($event)">
       <img src="/assets/Sun.png">
+      <br/>
+      <button (click)="calculateRange($event)">Calculate</button>
+      <br/>
+      <h1 *ngIf="(this.batteryError + this.speedError) == 0">
+        {{"The predicted range of the Eylsia is: " +range +" km"}}
```
Comment (JensVarughese) [inline Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html:22]: missing a space between the "+" and "range" on line 22

**Review by JensVarughese — APPROVED** [review]
Looks good from what I can see. Just wait until Jayden reviews this as well

File: Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html [hunk 2]
```diff
@@ -4,15 +4,31 @@ <h1> Elysia Range Simulator </h1>
     <form name="simulator" #simulator="ngForm" novalidate>
       <label>Speed (km/h):</label>
       <br>
-      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)">
-      <br>
+      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)" max=90 min=0>
+      <div *ngIf="speedError == 1; else speedError2" class="errorMessage">Speed is required</div>
+      <br/>
```
Comment (JaydenBischoff) [inline Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html:9]: The closing tag for a page break is not needed in HTML so you can remove the slash from the tag.

**Review by JaydenBischoff — CHANGES_REQUESTED** [review]
It looks really good so far. Right now, an error message will only show once the user has clicked the calculate button. Can you change this so that the error message shows when the user clicks outside of the input field and the error messages should be hidden when the user clicks on the input field.

**Pull request closed without merging**